modules/ROOT/pages/policy-mule4-tokenization.adoc

```diff
@@ -21,29 +21,37 @@ endif::[]
 
 |===
 
-[NOTE]
---
-This policy is designed to work only for Anypoint Runtime Fabric deployments.
---
+Tokenization is the process of tokenizing a value or piece of information, which might be considered sensitive data, into the token that can be mapped back to the original value. 
 
-== Tokenization
+This tokenized information replaces the original value with the token, thus minimizing the risk of matching it to the absolute value, in case a third party accesses the data. For example, this policy may be used to tokenize credit card numbers and map them to completely different ones from the originals.
```

Comment:
is absolute correct in this context? should be original.

Reply:
I would add a paragraph on why would you use this policy in the policy explanation before the examples, for instance:

If your API contains sensitive data, the tokenization policy is a highly effective way to protect it. Examples of sensitive information that are suitable for tokenization protection include credit card numbers, Personally Identifiable Information (PII), Protected Health Information (PHI), or any information deemed sensitive

Reply:
Done! Thanks!